src/sdk/strategy/cbd-strategy.ts

```diff
@@ -90,14 +90,18 @@ export class CbdStrategy {
 }
 
 export class DeployedCbdStrategy {
-  constructor(
-    public cohort: Cohort,
-    public dkgRitual: DkgRitual,
-    public encrypter: Enrico,
-    public decrypter: CbdTDecDecrypter,
+  public constructor(
+    public readonly cohort: Cohort,
+    public readonly dkgRitual: DkgRitual,
+    public readonly encrypter: Enrico,
+    public readonly decrypter: CbdTDecDecrypter,
```

Comment:
Can the decrypter simply be made by using the `cohort.configuration`? If so, do we need to pass it in the contstructor?

Looking at this piece of code:
```typescript
const decrypter = new CbdTDecDecrypter(cohort.configuration.porterUri);
    return new DeployedCbdStrategy(
      cohort,
      ritual,
      encrypter,
      decrypter,
      maybeConditionSet
    );
```